README.md

```diff
@@ -80,13 +81,80 @@ In the end the PCB will not be visible from the outside, powered from the inside
+        Type string : CupAvailable   "Tasse vorh."      [stateTopic="devices/senseo-wifi-home/machine/cupAvailable"]
+        Type string : CupFull        "Tasse voll"       [stateTopic="devices/senseo-wifi-home/machine/cupFull"]
+        Type string : Online         "Online"           [stateTopic="devices/senseo-wifi-home/$online"]
+
+        Type string : version        "Version"          [ stateTopic="devices/senseo/$homie"]
+        Type string : name           "Name"             [ stateTopic="devices/senseo/$name"]
+        Type string : ip             "IP-address"       [ stateTopic="devices/senseo/$localip"]
+        Type string : mac            "MAC-address"      [ stateTopic="devices/senseo/$mac"]
+        Type string : fw_name        "Firmware Name"    [ stateTopic="devices/senseo/$fw/name"]
+        Type string : fw_version     "Firmware Version" [ stateTopic="devices/senseo/$fw/version"]
+        Type string : nodes          "Nodes"            [ stateTopic="devices/senseo/$nodes"]
+        Type string : implementation "Implementation"   [ stateTopic="devices/senseo/$implementation"]
+        Type string : interval       "Update interval"  [ stateTopic="devices/senseo/$stats/interval"]
+    }
+}
+```
+
+### SenseoWifi.items
+[expire binding](https://www.openhab.org/addons/bindings/expire1/) recommended for debug-value
+```
+String KU_Senseo_Debug         "Debug [%s]"                          {channel="mqtt:topic:SenseoWiFi:Debug", expire="10s"}
+String KU_Senseo_OpState       "Zustand [MAP(senseo-wifi.map):%s]"   {channel="mqtt:topic:SenseoWiFi:OpState"}
+Switch KU_Senseo_OnOff         "OnOff"                               {channel="mqtt:topic:SenseoWiFi:OnOff"}
+String KU_Senseo_Brew          "Brew"                                {channel="mqtt:topic:SenseoWiFi:Brew"}
+Number KU_Senseo_BrewedSize    "BrewedSize"                          {channel="mqtt:topic:SenseoWiFi:BrewedSize"}
+String KU_Senseo_OutOfWater    "OutOfWater"                          {channel="mqtt:topic:SenseoWiFi:OutOfWater" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_Recipe        "Recipe"                              {channel="mqtt:topic:SenseoWiFi:Recipe"}
+String KU_Senseo_CupAvailable  "Tasse vorh."                         {channel="mqtt:topic:SenseoWiFi:CupAvailable" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_CupFull       "Tasse voll"                          {channel="mqtt:topic:SenseoWiFi:CupFull" [profile="transform:MAP", function="senseo-wifi.map"]}
+String KU_Senseo_Online        "Online"                              {channel="mqtt:topic:SenseoWiFi:Online"}
+```
+
+### SenseoWifi.sitemap
+```
+sitemap testing label="Senseo-Tests"
+{
+    Frame label="Status" {
+        //Text item=KU_Senseo_Unreach label="Fehler [Maschine nicht verfügbar!]" visibility=[KU_Senseo_Online == "false"] valuecolor=["true"="red"]
```

Comment:
Is this line supposed to be commented out?

Reply:
Did not see that.
I'll commit some rework for the sitemap.